huaweicloud/services/acceptance/acceptance.go

```diff
@@ -197,6 +199,13 @@ func TestAccPreCheckMigrateEpsID(t *testing.T) {
 	}
 }
 
+// lintignore:AT003
+func TestAccPreCheckDNS(t *testing.T) {
```

Comment:
can we remove this specific check for DNS, not sure why DNS need to be tested differently.

Reply:
It seems like this specific check is useless. I will changed it with `TestAccPreCheck`